lib/query/country_divisions.rb

```diff
@@ -12,8 +12,8 @@ def initialize(id:)
 
     def data
       division_results.map do |r|
-        DivisionStruct.new(*r.values_at(:item, :population, :legislature, :office, :head))
-      end
+        DivisionStruct.new(*r.values_at(:item, :population, :office, :head), legislatures(r[:item]))
+      end.uniq(&:id)
```

Comment:
I may be mis-reading this, but isn't this going to throw away data when somewhere has, for example, co-heads of government? (e.g. Northern Ireland)

[We don't see to have pull request previews turned on on this repo, so it's difficult to validate that — @sagepe, is there any reason we can't turn these on here?]

Reply:
Yes, I think you're right. I included it because I was trying to fix another thing...

>It also ensures only one of each place is displayed on division and city lists. This problem was additionally noted on #8 (comment).

...but that should clearly be part of a separate issue and pull request. A related issue is #22 - fixing the bigger problem behind that will address this. I'll remove ac749ea4495cfb39e5986cb128b39164991851d2, which includes this change, from this pull request.